client/src/utils/getFirstCharacterFromString.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns the first character from a string
+ * @param {String} str Text from which the first character will be extracted
+ * @returns {String} First character from the string provided
+ */
+
```

Comment:
Remove this extra line? I'm not sure if JSDoc would be able to identify the function this comment is linked to if there's a line break in between.
```suggestion
```